setup.py

```diff
@@ -39,7 +39,7 @@
     include_package_data=True,
     install_requires=[
         'pycryptodomex', 'requests', 'lxml', 'future', 'mini-amf',
-        'attrs >= 18.1.0, < 18.3.0', 'ConfigArgParse >= 0.13.0'
+        'attrs >= 18.1.0, < 19.2.0', 'ConfigArgParse >= 0.13.0'
```

Comment:
Is there a good reason for the upper limit? How about removing it? 
```suggestion
        'attrs >= 18.1.0', 'ConfigArgParse >= 0.13.0'
```

Reply:
> Is there a good reason for the upper limit? How about removing it?
> Suggested change
>         'attrs >= 18.1.0, < 19.2.0', 'ConfigArgParse >= 0.13.0'
>         'attrs >= 18.1.0', 'ConfigArgParse >= 0.13.0'

According to aajanki, the API changes every few versions https://github.com/aajanki/yle-dl/issues/163#issuecomment-418457066

Reply:
Ha, I asked the same question back then :)